soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3447,6 +3448,11 @@ void DrawRandoEditor(bool& open) {
                                         "If disabled, only the Claim Check will be found in the pool.");
                     PaddedSeparator();
 
+                    SohImGui::EnhancementCheckbox(Settings::ShuffleMagicBeans.GetName().c_str(), "gRandomizeShuffleBeans");
```

Comment:
Not a problem with this PR, but @aMannus any thoughts/plans on the continually growing middle column?

Reply:
I did notice it, but not a whole lot we can do about it, as they really are all shuffle options (except nighttime GS).

Reply:
We could go with the idea Briaguya had a little while ago, adding the ability to have scrollbars in each column seperately. 

![image](https://user-images.githubusercontent.com/4244591/186844481-51e8dd47-8eca-4f41-9bcf-3666ac8f6dcd.png)

Maybe better to have this discussion somewhere else though.

Reply:
Discussion on this should probably continue here https://github.com/HarbourMasters/Shipwright/pull/1321